src/DataFramesMeta.jl

```diff
@@ -44,6 +44,90 @@ replace_syms!(e::Expr, membernames) =
         mapexpr(x -> replace_syms!(x, membernames), e)
     end
 
+"""
+    @col(kw)
+
+`@col` transforms an expression of the form `z = :x + :y` into it's equivalent in
```

Comment:
maybe add `[:x, :y] => ((x,y) -> +(x,y)) => :z` (or whatever it exactly is) and then say that it is DataFrames.jl syntax as you do later in this sentence.

Reply:
oh - I see that you have added in the example. Then decide if to duplicate it or leave as is.

Reply:
Leaving as is.